src/main/java/edu/harvard/iq/dataverse/dataset/DatasetType.java

```diff
@@ -53,6 +54,12 @@ public class DatasetType implements Serializable {
      */
     @ManyToMany(cascade = {CascadeType.MERGE})
     private List<MetadataBlock> metadataBlocks = new ArrayList<>();
+    
+    /**
+     * The Licenses this dataset type is linked to.
+     */
+    @ManyToMany(cascade = {CascadeType.MERGE})
+    private List<License> licenses = new ArrayList<>();
```

Comment:
Don't we need a Flyway script for this?

Reply:
You're right. I just deployed develop, kept my database, and deployed this branch, and it's fine.